test/bulk2.test.ts

```diff
@@ -225,28 +225,50 @@ if (isNodeJS()) {
     }
   });
 
-  it('should bulk delete from file and return deleted results', async () => {
-    const id = Date.now();
+  it('should bulk insert from file and return inserted results', async () => {
+    // insert 100 account records from csv file
+    const csvStream = fs.createReadStream(
+      path.join(__dirname, 'data', 'Account_bulk2_test.csv'),
+    );
 
-    await insertAccounts(id);
+    const res = await conn.bulk2.loadAndWaitForResults({
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
why not `require` at the top of the file?

Reply:
e2e test files are loaded in node and browser (via webpack <-> karma setup) so we can't have top-level, node-only code here.
It's safe to do the import here because this test block only runs on node:
https://github.com/jsforce/jsforce/blob/0bbf6313628ff5380138ea5df23b06a59d4fe58c/test/bulk2.test.ts#L196

failure with top-level import:
https://github.com/jsforce/jsforce/actions/runs/11902422107/job/33167424899